components/climate/thermostat.rst

```diff
@@ -422,6 +378,67 @@ experience and automation.
         preset_change:
           - logger.log: Preset has been changed!
 
+Default Preset
+**************
+
+These configuration items determine default values the thermostat controller should use when it starts.
+
+- **default_preset** (*Optional*, string): The name of the preset to use by default. Must match a preset
+  as per :ref:`preset <thermostat-preset>`.
+- **on_boot_restore_from**: (*Optional*, on_boot_restore_from): Controls what the thermostat will do when
+  it first boots. One of:
+
+    - ``memory`` (default): The thermostat will restore any settings from last time it was running.
+    - ``default_preset``: The thermostat will always switch to the presets specified by **default_preset**
```

Comment:
Total nitpick here, but this should probably say "preset" (singular) and not "presets". Could be confusing. 😇 You'd never switch to more than one preset (unless there is something going on that I don't know about...lol).

Reply:
Good catch, fixed!